packages/react/src/components/Video/VideoNative.tsx

```diff
@@ -0,0 +1,52 @@
+import "react-native-get-random-values";
+import { WebRTC } from "react-native-webrtc";
+import VideoBase from "./VideoBase";
+import { IVideoCommonProps } from "./IVideoCommonProps";
+import { Video } from "@signalwire/js";
+import { useState } from "react";
+interface IVideoNativeProps extends IVideoCommonProps {
+  rtcViewStyle: React.CSSProperties;
+}
+
+const VideoNative: React.FC<IVideoNativeProps> = ({
+  token,
+  onRoomUpdate = () => {},
+  onEvent = (eventName, event) => {
+    console.log(eventName, event);
+  },
+  rtcViewStyle = {
+    top: 0,
+    position: "absolute",
+    height: "100%",
+    width: "100%",
+  },
+  ...props
+}) => {
```

Comment:
This file should go in the react-native repo: https://github.com/signalwire-community/react-native

Reply:
after we push the web version, you will be able to import it as a package dependency in RN